### PR TITLE
ci: auto mark issue reply status

### LINKED ISSUE #475: [CI] auto add issues comment reply status
### Description

mark the status of the issue as pending handle (contributor should reply) or pending feedback (issue author should reply) like this

image

Reference: https://github.com/dataease/dataease/issues

### PR DESCRIPTION
**Pull request by zhicheng-ning**
## Brief Information

This pull request is in the type of:

- [ ] bug fixing
- [ ] new feature
- [x] ci

Related issues:

- resolve #475 

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/hypertrons/hypertrons-crx/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/hypertrons/hypertrons-crx)

### CODE REVIEW

File: .github/workflows/issue-comment.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        uses: actions-cool/issues-helper@v2
+        with:
+          actions: 'remove-labels'
+          labels: 'comment/pending-feedback'
+
+  add-label-if-not-author:
+    runs-on: ubuntu-latest
+    if: (github.event.issue.user.id != github.event.comment.user.id) && !github.event.issue.pull_request && (github.event.issue.state == 'open')
+    steps:
+      - name: Add require replay label
```
Comment (tyn1998) [inline .github/workflows/issue-comment.yml:30]: tiny typo: "replay" -> "reply"

**Review by tyn1998 — CHANGES_REQUESTED** [review]

File: .github/workflows/issue-comment.yml (hunk 1 above)
Comment (zhicheng-ning) [reply .github/workflows/issue-comment.yml]: done

File: .github/workflows/issue-comment.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        uses: actions-cool/issues-helper@v2
+        with:
+          actions: 'add-labels'
+          labels: 'waiting for repliers'
+
+      - name: Remove require reply label
+        uses: actions-cool/issues-helper@v2
+        with:
+          actions: 'remove-labels'
+          labels: 'waiting for author'
```
Comment (tyn1998) [inline .github/workflows/issue-comment.yml:24]: Hi, @zhicheng-ning, can we do the remove-label step first than do the add-label step?

Does this influence how the event show in timeline?

image

Comment (zhicheng-ning) [reply .github/workflows/issue-comment.yml:24]: It doesn't influence the order the event show in timeline.

Comment (tyn1998) [reply .github/workflows/issue-comment.yml:24]: Got it, thank you for testing.

**Review by tyn1998 — APPROVED** [review]
LGTM. 

Thank you!

/approve

**Comment by tyn1998** [comment]
Oh I fogot that PRs that are related to workflows should be merged manually.

**Pull request merged**